.github/workflows/semantic-pr.yml

```
@@ -9,11 +9,14 @@ on:
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
+  cancel-in-progress: false
 
 permissions:
   pull-requests: read
 
+  permission to actions_185321: read 
```

actionlint (1.7.12)

[error] unknown permission scope "permission to actions_185321". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)